build.sbt

```diff
@@ -35,7 +36,7 @@ lazy val root = project
 
 publishArtifact := false
 
-lazy val sonatypePublish = sonatypeSettings ++ Seq(
+lazy val sonatypePublish = Seq(
```

Comment:
why you removed `sonatypeSettings` ?

Reply:
The newest version of sbt-sonatype doesn't seem to require that. Am I missing something?